internal/common/common.go

```diff
@@ -393,6 +393,16 @@ func GetTagFromImageRef(ref string) string {
 	}
 }
 
+func GetAPIVipDNSName(c *Cluster) string {
```

Comment:
Perhaps this function should be called "GetConvertedClusterAPIVipDNSName", because it assumes UMN is correctly set, which is only true during conversion